app/assets/stylesheets/common/components/buttons.scss

```diff
@@ -14,16 +14,22 @@
   $hover-icon-color: var(--primary-low)
 ) {
   @include form-item-sizing;
+  border: none; // removing border in order to use better active styles
```

Comment:
Thanks for looking over this. I was trying to avoid the issue where the gradient that darkens the background doesn't cover the top 1px and bottom 1px because of the transparent border. 

When looking at this on dev it isnt happening actually, but it was in the style guide. 🤔 

I will take another pass at this monday.